### PR TITLE
dogbreedclassifier

### PR DESCRIPTION
**Pull request by Architkumar13**
A simple project to classify dog breeds.

### CODE REVIEW

**Comment by Architkumar13** [comment]
can someone please this pull request.

**Comment by Architkumar13** [comment]
can someone please review this pul request

File: neural_network/DogBreed.ipynb [hunk 1]
```diff
@@ -0,0 +1,12847 @@
...
+            "visibility": null,
+            "align_self": null,
+            "height": null,
+            "min_height": null,
+            "padding": null,
+            "grid_auto_rows": null,
+            "grid_gap": null,
+            "max_width": null,
+            "order": null,
+            "_view_mo
```
Comment (poyea) [inline neural_network/DogBreed.ipynb:10355]: For imports, you may want to organize them in certain ways, like what's been suggested in [PEP8](https://www.python.org/dev/peps/pep-0008/#imports).

Comment (poyea) [inline neural_network/DogBreed.ipynb:10466]: seems unnecessary?

**Review by poyea — COMMENTED** [review]
Just some possible changes: 
* like I point out, since you're importing a lot of packages, you may want to reorganize them;
* what is the specific problem we're trying to address here? Try adding some comments / explanation to the notebook as a complement to each code block (I mean, make use of the notebook - typeset your comment outside the code) [It could be just some short texts, like this one](https://github.com/TheAlgorithms/Jupyter/blob/master/other/food_wastage_analysis_from_1961-2013_fao.ipynb)

* (optional) on the reference: You may want to include the origin of the problem (give a title & a description), (a stable) source of data (or the label of that dataset), and references (like methodology, any external sources referred) in your ipynb. All of these can be put in text section on the notebook.

Thanks for the work!

**Pull request closed without merging**